src/Mechanisms/HandleComponents/Synthesizers/Tests/EnumUnitTest.php

```diff
@@ -34,7 +35,7 @@ public function test_an_enum_can_be_validated()
         Livewire::test(ComponentWithValidatedEnum::class)
             ->call('save')
             ->assertHasErrors('enum')
-            ->set('enum', ValidatedEnum::TEST->value)
+            ->set('enum', ValidatedEnum::TEST)
```

Comment:
We need to change this back to the setting the value as a string, to ensure Livewire's Enum hydration actually functions. Setting to the Enum already will potentially skip that.

```suggestion
            ->set('enum', ValidatedEnum::TEST->value)
```